ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Github has a notice on this file:

" You are using an old version of issue templates. Please update to the new issue template workflow. Learn more "

https://docs.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates

Bob wallet has a good issue template file.

In addition I think issue and pr templates should go in a `.github` directory.

I would recomened TWO issue templates using the new github format:

1) start a proposal discussion - informal proposal with vague details or just a statement of a problem ("how can we sotre blinded bid values on chain in such a way that they can be recovered with bip44?") etc or ("using TXT records to store the name of your favorite movie") so the community can vet the idea at the very beginning, before the author wastes time doing research or writing code

2) an actual issue with an existing HIP or the repo itself ("HIP-1 links are broken") etc.